File: task2.js
//2 - Daxil edilən müsbət tam ədədin palindrom olub-olmamasını yoxlayın

let number = 121; 
let mod= 0;
let foundNumber = 0; 
for (let i=number; i>=1; i=i/10)
{
    mod = i%10;
    foundNumber = mod + foundNumber;
    if (i>=10) {foundNumber = foundNumber * 10; }}
if (foundNumber == number) { console.log("It is a palindrom number"); }
else { console.log("It is not a palindrom number"); }

File: task2.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ConsoleApp1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int number = 121; 
            int mod= 0;
            int foundNumber = 0; 
            for (int i=number; i>=1; i=i/10)
            {
                mod = i%10;
                foundNumber = mod + foundNumber;
                if (i>=10) {foundNumber = foundNumber * 10; }
            }
            if (foundNumber == number) { Console.WriteLine("It is a palindrom number");
            }
            else { Console.WriteLine("It is not a palindrom number"); }
        }
    }
}